src/DnsResolver/DnsResolver.Autorest/README.md

```diff
@@ -70,13 +67,14 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
-commit: 37072829f795ce840b8085035e8adf4721602f34
+commit: 6286ea80b3ac7eecfc2f230d65b2034a656f10bf
 require:
-  - $(this-folder)/../../readme.azure.noprofile.md
+  - $(this-folder)/../../SwaggerCI/readme.azure.noprofile.md
```

Comment:
This line should be the same with original

Reply:
Fixed thank you